### PR TITLE
fix: Support null article data on article package

### PR DESCRIPTION
**Pull request by tuncaulubilge**
Native apps can use the article package with no data to show an error view, so the article package should work when article is undefined. Added a default value and a test

### CODE REVIEW

File: packages/article/src/article.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ export const templates = {
 
 const Article = props => {
   const {
-    article: { template = "mainstandard" }
+    article: { template = "mainstandard" } = { template: "mainstandard" }
```
Comment (nathggns) [inline packages/article/src/article.js]: This syntax was ugly to begin with, and it doesn't actually add support for `null` articles. (null is considered a value by the default argument mechanism). Also, the default value is duplicated which is unnecessary 

It's probably time to just improve the syntax:

```
const { template = "mainstandard" } = props.article || {};
```

Comment (nathggns) [reply packages/article/src/article.js]: [Here's what I mean about the default argument mechanism](https://codepen.io/nathggns/pen/jQeYwr?editors=0010#0). It's using function arguments, but it applies to destructuring too.

Comment (tuncaulubilge) [reply packages/article/src/article.js]: That looks much better, but airbnb lint is not happy with is as it forces destructing props. I could do this:
```
  const { article = {} } = props;
  const { template = "mainstandard" } = article;
```

Comment (nathggns) [reply packages/article/src/article.js]: 🤦‍♂️  _remind me why we use a linter that enforces destructing? (Which often makes code less clear, not more)_ 

Yeah that's fine.

**Pull request merged**